CSS/basic-styling/color.css

```diff
@@ -6,10 +6,8 @@
     --danger-color: #FA2A1D;
     --success-color: #2EBF34;
     --warning-color: #FF9800;
```

Comment:
Hover over the component on screen and check the accessibility for each of the alert boxes.